pkg/vadmin/suite_test.go

```diff
@@ -119,6 +115,15 @@ func (m *MockVClusterOps) VerifyCommonOptions(options *vops.DatabaseOptions) err
 	return nil
 }
 
+// VerifyCommonOptions is used in vcluster-ops unit test for verifying the common options among all db ops
+func (m *MockVClusterOps) VerifyCommonOptions(options *vops.DatabaseOptions) error {
+	if *options.Name != TestDBName {
```

Comment:
In the future, we will have db name in vcluster fetchNodeState. I think we can just simply assign the dbname to opts.Name in genFetchNodeStateOptions(). Then we don't need to change VerifyCommonOptions() here.